File: level_1/2016년.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

char* solution(int a, int b) {
    int dic[13] = {31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
    char    week[7][3] = {"THU", "FRI", "SAT", "SUN", "MON", "TUE", "WED"};
    int total_days = 0;
    for (int i = 0; i < a - 1; i++)
        total_days += dic[i];
    total_days += b;
    char* answer = (char*)malloc(sizeof(char) * 3);
    for (int j = 0; j < 3; j++)
        answer[j] = week[total_days % 7][j];
    answer[3] = '\0';
    return answer;
}

File: level_1/2016년.java
public class Solution {
    public String solution(int a, int b) {
        String[] weeks = { "FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU" };
        int[] months = { 31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 };
        int total_days = -1;
        for (int i = 0; i < a - 1; i++)
            total_days += months[i];
        total_days += b;
        return weeks[(total_days % 7)];
    }
}
